libr/bin/format/mach0/mach0.c

```diff
@@ -3227,44 +3220,61 @@ RSkipList *MACH0_(get_relocs)(struct MACH0_(obj_t) *bin) {
 			return NULL;
 		}
 
-		for (ur.p = opcodes, end = opcodes + amount ; ur.p < end; ) {
-			ut8 imm = *ur.p & BIND_IMMEDIATE_MASK, op = *ur.p & BIND_OPCODE_MASK;
-			++ur.p;
-			if (ur.p >= end) {
-				break;
-			}
-			switch (op) {
+		int partition_sizes[] = {bind_size, lazy_size, weak_size}, partition_idx;
+		int opcodes_offset = 0;
```

Comment:
this variable is never modified. can be removed

Reply:
It's modified at the end of the loop.